app/components/mfa_selection_component.html.erb

```diff
@@ -0,0 +1,24 @@
+<lg-validated-field>
+  <%= check_box_tag(
+        'two_factor_options_form[selection][]',
+        option.type,
+        option.type === 'phone' && flash[:phone_error].present?,
+        disabled: option.disabled?,
+        class: 'usa-checkbox__input usa-checkbox__input--tile',
+        id: "two_factor_options_form_selection_#{option.type}",
+      ) %>
+  <%= label_tag(
+        "two_factor_options_form_selection_#{option.type}",
+        class: "#{'checkbox__invalid' if option.type === "phone" && flash[:phone_error]} usa-checkbox__label usa-checkbox__label--illustrated",
```

Comment:
this might be clearer as an array instead of a string?
```suggestion
        class: [
          option.type === "phone" && flash[:phone_error] && 'checkbox__invalid',
          'usa-checkbox__label',
          'usa-checkbox__label--illustrated',
        ].select(&:present?).join(' '),
```